app/views/allocate_prison_offender_managers/_pom_tables.html.erb

```diff
@@ -20,23 +20,25 @@ Recommendation:
     </tr>
   </thead>
   <tbody class="govuk-table__body">
-      <tr class="govuk-table__row">
-        <td aria-label="POM name" class="govuk-table__cell table_cell__left_align poms-table-cell__key">Green, Samantha</td>
-        <td aria-label="Previous allocation" class="govuk-table__cell">-</td>
-        <td aria-label="Tier A cases" class="govuk-table__cell table_cell__left_align">11</td>
-        <td aria-label="Tier B cases" class="govuk-table__cell table_cell__left_align">10</td>
-        <td aria-label="Tier C cases" class="govuk-table__cell table_cell__left_align">2</td>
-        <td aria-label="Tier D cases" class="govuk-table__cell table_cell__left_align">1</td>
-        <td aria-label="Total" class="govuk-table__cell table_cell__left_align">24</td>
-        <td aria-label="Working pattern" class="govuk-table__cell">Full time</td>
-        <% if current_page?(action: 'show') %>
-          <td aria-label="Action" class="govuk-table__cell table_cell__left_align"><a
-                                href="#">Allocate</a></td>
-        <% else %>
-          <td aria-label="Action" class="govuk-table__cell table_cell__right_align"><a
-                              href="#">Reallocate</a></td>
-        <% end %>
-      </tr>
+  <% @poms.each do |pom| %>
+    <tr class="govuk-table__row">
+      <td aria-label="POM name" class="govuk-table__cell "><%= pom.full_name %></td>
+      <td aria-label="Previous allocation" class="govuk-table__cell">-</td>
+      <td aria-label="Tier A cases" class="govuk-table__cell "><%= pom.tier_a %></td>
+      <td aria-label="Tier B cases" class="govuk-table__cell "><%= pom.tier_b %></td>
+      <td aria-label="Tier C cases" class="govuk-table__cell "><%= pom.tier_c %></td>
+      <td aria-label="Tier D cases" class="govuk-table__cell "><%= pom.tier_d %></td>
+      <td aria-label="Total" class="govuk-table__cell "><%= pom.total_cases %></td>
+      <td aria-label="Working pattern" class="govuk-table__cell"><%= pom.working_pattern %></td>
+      <% if current_page?(action: 'show') %>
```

Comment:
I think these are the wrong way around, but we can fix this when we do the actual allocation.

Reply:
They are; I missed this when we changed the route/action from edit to show in PR#91